docs/modules/ROOT/pages/create.adoc

```diff
@@ -0,0 +1,71 @@
+== How to Create a Starter Kit
+
+=== What is a Starter Kit?
```

Comment:
I'd drop this section, since this is already covered previously in the docs. Do add something like this if you share this on the forum, though.

Instead, I'd add an intro paragraph (right below the main title) explaining what this doc is about: _in this guide you'll learn to create your own starter kit, read more about them here (link to getting started)_, or similar. Conversely, I would not include this paragraph in a posting on the forum, just to the documentation.